src/app-layout/visual-refresh/context.tsx

```diff
@@ -14,7 +14,6 @@ import { applyDefaults } from '../defaults';
 import { AppLayoutContext } from '../../internal/context/app-layout-context';
 import { DynamicOverlapContext } from '../../internal/context/dynamic-overlap-context';
 import { AppLayoutProps } from '../interfaces';
-import { DrawerItemProps, DrawersProps, InternalDrawersProps } from './drawers';
```

Comment:
There was a duplicate type between refresh and classic. It also contributed to the issue with `drawers?.onResize` above.

Removed duplicate types, used types from classic in both. (It would be a good next step to move the types into `src/app-layout/interfaces.ts` file)

Reply:
Yeah, that is my plan after this. I'll consolidate as much of the code as possible when bringing it out of beta.